documentation/writing_plans.md

```diff
@@ -26,6 +26,34 @@ extension. For example, given a plan named `my_plan.pp` in a module named
 `my_module`, the location of the plan would be
 `site-modules/my_module/plans/my_plan.pp`.
 
+## Creating a new project-level Puppet language plan
+
+You can create a new project-level Puppet language plan in your Bolt project
+using a Bolt command. The command accepts a single argument: the name of the
+plan. Project-level plans must be namespaced to the project.
+
+_\*nix shell command_
```

Comment:
Does this render for you? It isn't for me...
![unicode](https://user-images.githubusercontent.com/5055819/100026688-1a0cdd00-2da0-11eb-922d-a37ddfc2302f.png)

Reply:
Ya, renders just fine.

![image](https://user-images.githubusercontent.com/30373575/100026904-7c65dd80-2da0-11eb-81a7-81f5710bb69f.png)